tests/integration/remote/invoke/remote_invoke_integ_base.py

```diff
@@ -0,0 +1,105 @@
+from unittest import TestCase, skipIf
```

Comment:
This will be rebased with the class in the other PR?

Reply:
Yes that's right. I will cherry-pick and sync all the PRs with any new changes (`RemoteInvokeIntegBase` class) and rebase this PR after merging the other ones.